darts/utils/likelihood_models.py

```diff
@@ -1,32 +1,150 @@
 """
 Likelihood Models
 -----------------
+
+The likelihood models contain all the logic needed to train and use Darts' neural network models in
+a probabilistic way. This essentially means computing an appropriate training loss and sample from the
+distribution, given the parameters of the distribution.
+
+By default, all versions will be trained using their negative log likelihood as a loss function
+(hence performing maximum likelihood estimation when training the model).
+However, most likelihoods also optionally support specifying time-independent "prior"
+beliefs about the distribution parameters.
+In such cases, the a KL-divergence term is added to the loss in order to regularise it in the
+direction of the specified prior distribution. (Note that this is technically not purely
+a Bayesian approach as the priors are actual parameters values, and not distributions).
+The parameter `prior_strength` controls the strength of the "prior" regularisation on the loss.
+
+Some distributions (such as ``GaussianLikelihood``, and ``PoissonLikelihood``) are univariate,
+in which case they are applied to model each component of multivariate series independently.
+Some other distributions (such as ``DirichletLikelihood``) are multivariate,
+in which case they will model all components of multivariate time series jointly.
+
+Univariate likelihoods accept either scalar or array-like values for the optional prior parameters.
+If a scalar is provided, it is used as a prior for all components of the series. If an array-like is provided,
+the i-th value will be used as a prior for the i-th component of the series. Multivariate likelihoods
+require array-like objects when specifying priors.
+
+The target series used for training must always lies within the distribution's support, otherwise
+errors will be raised during training. You can refer to the individual likelihoods' documentation
+to see what is the support. Similarly, the prior parameters also have to lie in some pre-defined domains.
```

Comment:
```suggestion
to see what target values it supports. Similarly, the prior parameters also have to lie in some pre-defined domains.
```?

Reply:
Here too I'd rather keep the technical term (since it's also referred to later on).